tests/test_serialization.py

```diff
@@ -42,6 +42,7 @@
 from msrest.serialization import Model, last_restapi_key_transformer, full_restapi_key_transformer, rest_key_extractor
 from msrest import Serializer, Deserializer
 from msrest.exceptions import SerializationError, DeserializationError, ValidationError
```

Comment:
`SerializationError` and `DeserializationError` definition is removed by the PR so we should remove it?